Infrastructure/Infrastructure/ProgressTrackers/ExcelImporter.cs

```diff
@@ -74,7 +74,7 @@ private async Task<Unit> InsertTable(DataTable table)
                     command = new InsertPlotsCommand() 
                     { 
                         Table = table,
-                        IncrementProgress = OnIncrementProgress
+                        Progress = Progress
```

Comment:
??

Reply:
ExcelImporter has a property called Progress. That object needs to be passed into the query to track progression, so InsertPlotCommand also has a property called Progress which references the original Progress through this assignment.

Reply:
one of these names has to change

Reply:
This is one of those github changes situations where it only shows me a small piece and 
I didn't take enough time work out what it was doing ie: passing down local properties to children - which is still ugly but it probably didn't need to be changed - I needed to take a look at the bigger picture. 

Weren't commands using DI - or does it need a specific instance of the progress tracker?

While changing the name of ProgressTracker to TaskRunner does make it sound more like an 'is-a' relationship - if all that it is doing is tracking progress then the original name is probably more suitable but should be changed to a 'has-a' relationship? What do you think?